_includes/page__hero.html

```diff
@@ -1,20 +1,6 @@
-{% if page.header.image contains "://" %}
-  {% capture img_path %}{{ page.header.image }}{% endcapture %}
-{% else %}
-  {% capture img_path %}{{ page.header.image | relative_url }}{% endcapture %}
-{% endif %}
-
-{% if page.header.cta_url contains "://" %}
-  {% capture cta_path %}{{ page.header.cta_url }}{% endcapture %}
-{% else %}
-  {% capture cta_path %}{{ page.header.cta_url | relative_url }}{% endcapture %}
-{% endif %}
-
-{% if page.header.overlay_image contains "://" %}
-  {% capture overlay_img_path %}{{ page.header.overlay_image }}{% endcapture %}
-{% elsif page.header.overlay_image %}
-  {% capture overlay_img_path %}{{ page.header.overlay_image | relative_url }}{% endcapture %}
-{% endif %}
+{% capture img_path %}{{ page.header.image | relative_url }}{% endcapture %}
```

Comment:
Think these captures can all be removed and just use the page variables directly. `{{ page.header.cta_url | relative_url }}`, `{{ page.header.overlay_image | relative_url }}`, etc.

Reply:
For these three captures I removed the first two. The last one `overlay_img_path` is used multiple times later so I find it better to be kept.